src/engine/enginemaster.cpp

```diff
@@ -361,6 +361,10 @@ void EngineMaster::processChannels(int iBufferSize) {
         }
     }
 
+    // Do internal master sync post-processing before the other
+    // channels.
+    m_pMasterSync->onCallbackEnd(m_iSampleRate, m_iBufferSize);
```

Comment:
the correct name might be postProcess, to match the other decks.  With this change, the ordering is now:

InternalClock Begin
Decks Begin
InternalClock End
Decks End

Whereas before, the internal clock was updated after all of the decks, and this caused problems because the decks need updated internalclock information when they post-process

Reply:
Ah I see, the function has moved into processChannels(). I missed that during my first review. 
Maybe we should move m_pMasterSync->onCallbackStart(m_iSampleRate, m_iBufferSize);
as well for code symmetric?